### PR TITLE
Bug/65067 ps report utf8 chars

### PR DESCRIPTION
**Pull request by jon-shipley**
Fix issue where accented characters were not correct in the psychometric report.

- Changed the bulk load CSV for the psychometric report to be UTF16LE as UTF8 is not supported on linux to ensure parity between dev and prod environments. 
- Prepended a UTF 16LE  BOM to the CSV file which seems essential for SQL Server. 
- Added metadata to the file for cloudops to see that this is a csv file, with utf16le encoding.

### CODE REVIEW

File: admin/app.js [hunk 1]
```diff
@@ -191,7 +191,7 @@ if (!config.Redis.useTLS) {
   logger.warn('redis running in non-secure mode')
 }
 
-const RedisStore = require('connect-redis')(session)
+const RedisStore = require('connect-redis').default
```
Comment (GuyHarwood) [inline admin/app.js:194]: what is the reason for this change?

Comment (jon-shipley) [reply admin/app.js:194]: `redis-connect` v6 upgraded to v7

**Review by GuyHarwood — APPROVED** [review]
LGTM

File: tslib/src/functions-ps-report/ps-report-3b-stage-csv-file/csv-transformer.ts [hunk 2]
```diff
@@ -3,6 +3,42 @@ import * as CSV from 'csv-string'
...
+  FormID: string | null
+  TestDate: string | null
+  TimeStart: string | null
+  TimeComplete: string | null
+  TimeTaken: number | null
+  ReasonNotTakingCheck: string | null
+  ToECode: number
+  ImportedFromCensus: number
+  IsEdited: number
+  // TODO: add fields for Q1-25 - the bulk of the structure
```
[the remark this answers is not in the document]
Comment (jon-shipley) [reply tslib/src/functions-ps-report/ps-report-3b-stage-csv-file/csv-transformer.ts:39]: Yes.
